### PR TITLE
🌱 docs: document the process

### PR DESCRIPTION
**Pull request by ncdc**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+## File structure
+
+All documentation-related items live in `docs` (with the small exception of various `make` targets and some helper 
+scripts in `hack`).
+
+The structure of `docs` is as follows:
+
+| Path                         | Description                                                                       |
+|------------------------------|-----------------------------------------------------------------------------------|
+| config/<language>/mkdocs.yml | Language-specific `mkdocs` configuration.                                         |
```
Comment (nrb) [inline docs/README.md:24]: This omitted the language in GH preview. `config//mkdocs.yml` is what I see.

Comment (nrb) [reply docs/README.md:24]: Same with `content/`
Screen Shot 2023-02-16 at 4 02 41 PM

Comment (ncdc) [reply docs/README.md:24]: thx fixing

**Review by vincepri — APPROVED** [review]

**Pull request merged**